src/helper/object.js

```diff
@@ -118,16 +117,32 @@ function toCamelCase(object, exceptions) {
   }, {});
 }
 
+function getLocationFromUrl(href) {
```

Comment:
Seems like this function might require to add a couple of test cases to verify that it's working fine.

Reply:
added a few of them

Reply:
👍